articles/connections/apple-siwa/set-up-apple.md

```diff
@@ -0,0 +1,125 @@
+
+![Developer Program](/media/articles/connections/social/apple/apple-developerprogram.jpg)
+:::
+
+After you register your application, you will be given the following IDs and keys to use in the application connection settings in the Dashboard:
+
+  * **App ID**
+  * **Services ID** (Client ID)
+  * **Apple Team ID**
+  * **Client Secret Signing Key**
+  * **Client Signing Key ID** (optional)
+
+## Prerequisite
+
+A paid [Apple Developer](https://developer.apple.com/programs/) account. 
+
+## Obtain Team ID
+
+1. Sign in to your [Apple Developer Account](https://developer.apple.com/account/#/overview/).
+
+2. Go to the [Membership page](https://developer.apple.com/account/#/membership/) of your Apple Developer account.
+
+    ![Membership Page](/media/articles/connections/social/apple/apple-membership.jpg)
+
+3. Make note of your Team ID.
+
+## Create App ID
+
+1. On the Apple Developer Portal, go to **Certificates, IDs, & Profiles > Identifiers** and click the **blue plus icon** next to **Identifiers** to create a new App ID.
+2. Choose **App IDs** as the identifier type and click **Continue**
+3. Provide a description and a Bundle ID (reverse-domain name style, e.g., `com.customdomain`)
+4. Scroll down and check **Sign In with Apple**. 
+5. Click **Continue** and then click **Register**
+
+## Create Services ID
+
+1. Back in the **Certificates, IDs, & Profiles** section, click the **blue plus icon** next to **Identifiers**.
+
+    ![Register Services ID](/media/articles/connections/social/apple/apple-registerservicesid.jpg)
+
+2. Choose **Services IDs** and click **Continue**. Fill in the description and identifier (`com.customdomain.webapp`).
+3. After checking **Sign In with Apple**, click on **Configure** and define your **Web Domain** (`customdomain.com`) and your **Return URL**. Make sure that your Return URL follows this format: `https://customdomain.com/login/callback`.
+
+    ![Configure URLs](/media/articles/connections/social/apple/apple-configureurls.jpg)
+
+4. Click **Save**, **Continue**, and then click **Register**.
+
+### Verify domain with Apple
+
+1. On the **Certificates, IDs, & Profiles** page, click your newly created Services ID.
+2. Click the **Configure** button next to the **Sign In with Apple** feature.
+
+3. On the **Redirect URI/Domain Validation** page click **Download**.
+4. Copy or move the `apple-developer-domain-association.txt` file to your application server and make it accessible at `https://customdomain.com/.well-known/apple-developer-domain-association.txt`.
+1. When this is ready, click **Verify** on your Services ID configuration page. If no error message is shown, your domain has been successfully verified.
+
+## Set up Signing Key
```

Comment:
This section should only make the developer download the private key from apple, copy its ASCII content to clibpoard and paste it in the manage dashboard. No JWT signing to get the secret value, thats all handled by the connection in auth0 using the private key.

Reply:
I believe Josh will be removing this section.

Reply:
Yeah, should be gone now.  Thanks 👍